src/collection.js

```diff
@@ -114,6 +127,48 @@ export default class Collection {
   }
 
   /**
+   * Validates an idSchema.
+   *
+   * @param  {Object} idSchema
+   * @return {Object}
+   */
+  _validateIdSchema(idSchema) {
+    if (!idSchema) {
```

Comment:
`if (typeof idSchema === "undefined") {` would be slightly more correct (although nobody should in practice specify `false` or `''` as their id schema, if we're checking the input then we might as well do it fully correctly).